os/arch/arm/src/armv7-a/arm_cpuflash.c

```diff
@@ -0,0 +1,115 @@
+/****************************************************************************
+ *
+ * Copyright 2023 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * arch/arm/src/armv7-a/arm_cpustart.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Do you get this from Nuttx? If not, it looks no need to add this license.

Reply:
No, this is a customisation for AIDual, we registered an additional software generated interrupt (SGI) to handle flash operation.

Reply:
done~